modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase

```diff
@@ -90,6 +90,90 @@ definition {
 		}
 	}
 
+	@description = "LPS-164582 - Verify CRUD functionality with audit when using object definition entry that has the entry history activated"
```

Comment:
Verify CRUD events appear in the Audit records for object entries when entry history is enabled